hammerdb/hammerd_wrapper.py

```diff
@@ -24,34 +24,38 @@ def _parse_stdout(stdout):
     data = []
     iteration = 0
     for line in stdout.splitlines():
+        print (line)
         if "TEST RESULT" in line:
             worker = (line.split(":"))[0]
             tpm = (line.split(" "))[6]
             nopm = (line.split(" "))[-2]
-            entry = [ worker, tpm, nopm]
+            timestamp = datetime.now()
```

Comment:
so I think we should be getting the timestamp just before we execute the hammerdb command vs while parsing the output